calendar-api/scripts/docker-test-image.sh

```
@@ -0,0 +1,72 @@
+  docker stop $container_id || :
+  docker network rm $network_name || :
+}
+
+trap cleanup EXIT
+
+curl() {
+  docker run -i --rm --network $network_id byrnedo/alpine-curl "$@"
+}
+
+poll_service_up() {
+  local url="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)